postgresql/scale-postgresql-server/scale-postgresql-server.sh

```diff
@@ -14,7 +14,7 @@ az postgres server create \
 --location westus \
 --admin-user myadmin \
 --admin-password <server_admin_password> \
---sku-name GP_Gen4_2 \
```

Comment:
L16:

__Recommended__: Use a generated password when creating a resource, and display it to the user at some point. See https://review.docs.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=master#randomize-passwords-for-new-resources